pylint_odoo/checkers/modules_odoo.py

```diff
@@ -484,6 +485,16 @@ def _get_duplicate_xml_fields(self, fields):
             all_fields.setdefault(field_xml, []).append(field)
         # Remove all keys which not duplicated
         for key, items in all_fields.items():
+            parents = [e.getparent() for e in items]
+            same_parent = [
+                item for item, count in collections.Counter(parents).items()
+                if count > 1]
```

Comment:
IMO a real loop is better than splitting generator logic on to multiple lines. Or maybe an intermediary variable for the iterator would suffice.